### DISCUSSION TITLE
Publishing theme to Open VSX?

### DISCUSSION THREAD

**Discussion by dasearle** [daltonmenezes/aura-theme#77] [Q&A] [answered]
Hi,

I love this theme for my local VS Code but have recently moved across to GitPod (gitpod.io) and it uses the Open VSX registry in place of the Visual Studio Marketplace. As the theme hasn't been published on the registry, I can't install it onto gitpod without breaking the Microsoft T&Cs on downloading and installing on non VS Code apps.

Could the theme be published onto the Open VSC registry (https://github.com/eclipse/openvsx/wiki/Publishing-Extensions)

Cheers

**Comment by daltonmenezes** [comment, marked as answer]
Hi @dasearle , can you try it and tell me if everything is working good please?

https://open-vsx.org/extension/DaltonMenezes/aura-theme

**Reply by dasearle** [reply]
Working great - able to install and use on GitPod. Many thanks!
